src/File.php

```diff
@@ -272,8 +273,8 @@ public function details() : array
             'checksum' => $this->checksum,
             'location' => $this->location,
             'file_path' => $this->filePath,
-            'created_at' => $now->format($this->cache::RFC_7231),
-            'expires_at' => $now->addSeconds($this->cache->getTtl())->format($this->cache::RFC_7231),
+            'created_at' => $now->format($cache::RFC_7231),
+            'expires_at' => $now->addSeconds($this->cache->getTtl())->format($cache::RFC_7231),
```

Comment:
@dmeybohm not that it really matters but why switch between using `$this->cache` and `$cache`?